app/controllers/management/budgets/investments_controller.rb

```diff
@@ -57,7 +58,10 @@ def load_investment_votes(investments)
     end
 
     def investment_params
-      attributes = [:external_url, :heading_id, :tag_list, :organization_name, :location, :skip_map]
+      attributes = [:external_url, :heading_id, :tag_list, :organization_name, :location, :skip_map,
+                    image_attributes: image_attributes,
+                    documents_attributes: [:id, :title, :attachment, :cached_attachment, :user_id, :_destroy],
```

Comment:
How about extracting these attributes to a method `documents_attributes` and share it across controllers, like it's done with `image_attributes`?

By the way, the `Legislation::ProposalsController` controller uses all these parameters except `:_destroy` :thinking:. Could you check whether it's done deliberately or accidentally? :pray: